### PR TITLE
ci: use npm workspaces

### LINKED ISSUE #287: 🤖 - Use npm workspaces
### Which package(s) are relevant/related to the feature request?

@maskito/core, @maskito/kit, @maskito/angular, @maskito/react

### Description

- [ ] Use [npm workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces).
- [ ] Refactor root `package.json`:
    * Move all react libraries to `projects/react/package.json`
    * Move all Taiga UI packages to `projects/demo/package.json`
    * ...

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Type

What kind of change does this PR introduce?

- [X] Build or CI related changes

Closes #287

### CODE REVIEW

File: projects/react/package.json [hunk 1]
```diff
@@ -29,9 +29,17 @@
...
+        "@types/react": ">=17.0.0",
+        "@types/react-dom": ">=17.0.0",
+        "react-test-renderer": ">=17.0.0"
+    },
     "peerDependencies": {
         "@maskito/core": "^0.11.0",
-        "react": ">=16.8",
-        "react-dom": ">=16.8"
+        "react": ">=17.0.0",
+        "react-dom": ">=17.0.0"
```
Comment (nextZed) [inline projects/react/package.json:43]: why?

Comment (waterplea) [reply projects/react/package.json:43]: @nsbarsukov I think we should keep the lowest possible version for greater compatibility.

Comment (nsbarsukov) [reply projects/react/package.json:43]: Yep, it is just an attempt to fix [new error](https://github.com/Tinkoff/maskito/actions/runs/4938046549/jobs/8827331876?pr=293).
I will revert it when i will find solution to it.

Comment (nsbarsukov) [reply projects/react/package.json]: fixed

**Review by splincode — APPROVED** [review]

**Review by nextZed — APPROVED** [review]

**Pull request merged**